tempodb/backend/raw.go

```diff
@@ -22,9 +26,7 @@ type KeyPath []string
 // RawWriter is a collection of methods to write data to tempodb backends
 type RawWriter interface {
 	// Write is for in memory data.  It is expected that this data will be cached.
-	Write(ctx context.Context, name string, keypath KeyPath, buffer []byte) error
-	// StreamWriter is for larger data payloads streamed through an io.Reader.  It is expected this will _not_ be cached.
-	StreamWriter(ctx context.Context, name string, keypath KeyPath, data io.Reader, size int64) error
+	Write(ctx context.Context, name string, keypath KeyPath, data io.Reader, size int64) error
```

Comment:
This is the most important file to review in this change, all the other changes are about adapting the new interface definitions.